src/main/webapp/js/omniComment.js

```diff
@@ -220,7 +220,7 @@ $(document).ready(function() {
         
         // to show student comments (only works for Giver filter)
         if (commentToShow.hasClass('student-record-comments')) {
-            var studentCommentPanelBody = commentToShow.closest('.student-comments-panel').parent();
+            var studentCommentPanelBody = commentToShow.parent();
```

Comment:
I suspect this might cause some changes in behaviour, since the hide method uses `commentToShow.parent()` currently.

Switching both the show and hide methods to `commentToShow.closest('.panel-body')` will likely disallow the showing and hiding of individual instructor panels in the student comments section, but this can be left in if it's considered acceptable behaviour. I'll post a few screenshots later showing the difference.